fitlins/cli/run.py

```diff
@@ -19,12 +19,12 @@
 from argparse import RawTextHelpFormatter
 from multiprocessing import cpu_count
 
-from bids.layout import BIDSLayout
+import bids
```

Comment:
why not import both object instead of whole library? nitpick though otherwise lgtm

Reply:
My aesthetic preferences go through phases... bids is a short namespace, so seems cheap to be explicit.